callattendant/hardware/modem.py

```diff
@@ -281,6 +281,7 @@ def _call_handler(self, handle_caller):
 
                     if dev_mode:
                         logfile.write(modem_data.encode())
+                        logfile.write(os.linesep.encode())
```

Comment:
The original purpose of this logfile was to capture RAW output from the modem. The file is open in "binary" mode so things like line-endings, control characters, etc. are recorded. Adding arbitrary line-endings after the data subverts this process. In fact, I have been intending to remove this logging feature as it is no longer needed.